src/components/Profile/Profile.js

```diff
@@ -31,8 +30,27 @@ const useStyles = makeStyles((theme) => ({
 export default function Profile() {
   const classes = useStyles();
   const { userData } = useAuth();
```

Comment:
so then userData updates whenever theres a change in the db, like if we edit the profile? does it make the page re-render with the new info without needing a refresh?

Reply:
just tested it and looks good, should be good to merge